### PR TITLE
docs(peapods): add peapods docstrings

### PR DESCRIPTION
**Pull request by Yongxuanzhang**

### CODE REVIEW

File: jina/peapods/runtimes/asyncio/rest/app.py [hunk 1]
```diff
@@ -19,6 +19,13 @@
 
 
 def get_fastapi_app(args: 'argparse.Namespace', logger: 'JinaLogger'):
+    """
+    Get the app from FastAPI as the REST interface.
+
+    :param args: passed arguments.
+    :param logger: Jina logger.
+    :return:
```
Comment (theUnkownName) [inline jina/peapods/runtimes/asyncio/rest/app.py:27]: Is there a return here?

File: jina/peapods/runtimes/asyncio/rest/app.py [hunk 2]
```diff
@@ -195,7 +239,15 @@ async def on_connect(self, websocket: WebSocket) -> None:
             logger.success(f'Client {self.client_info} connected to stream requests via websockets')
 
         async def handle_receive(self, websocket: WebSocket, close_code: int) -> None:
+            """
+            Await a message on :meth:`websocket.receive()`
+            Send the message to zmqlet via :meth:`zmqlet.send_message()` and await
+
+            :param websocket: WebSocket connection between clinet sand server.
+            :param close_code: close code
+            """
```
Comment (theUnkownName) [inline jina/peapods/runtimes/asyncio/rest/app.py:248]: Maybe add what's returned here?

File: jina/peapods/runtimes/jinad/client.py [hunk 3]
```diff
@@ -16,19 +16,21 @@
 
 
 class DaemonClient:
+    """
+    Jina Daemon client.
+
+    :param host: the host address of ``jinad`` instance
+    :param port: the port number of ``jinad`` instance
+    :param logger:
```
Comment (theUnkownName) [inline jina/peapods/runtimes/jinad/client.py:24]: The `logger` has missing info

File: jina/peapods/runtimes/zmq/zed.py [hunk 4]
```diff
@@ -129,9 +130,9 @@ def _handle(self, msg: 'Message') -> 'ZEDRuntime':
         """Register the current message to this pea, so that all message-related properties are up-to-date, including
         :attr:`request`, :attr:`prev_requests`, :attr:`message`, :attr:`prev_messages`. And then call the executor to handle
         this message if its envelope's  status is not ERROR, else skip handling of message.
+
```
Comment (theUnkownName) [inline jina/peapods/runtimes/zmq/zed.py:133]: Maybe we can add also what's returned here?

**Review by theUnkownName — CHANGES_REQUESTED** [review]
Just small things, great job!

**Review by cristianmtr — DISMISSED** [review]

File: jina/peapods/runtimes/asyncio/rest/app.py (hunk 1 above)
Comment (Yongxuanzhang) [reply jina/peapods/runtimes/asyncio/rest/app.py:27]: No return. Thanks for picking out!

File: jina/peapods/runtimes/asyncio/rest/app.py (hunk 2 above)
Comment (Yongxuanzhang) [reply jina/peapods/runtimes/asyncio/rest/app.py:248]: Seems no return(None) for this function？

File: jina/peapods/peas/helper.py [hunk 5]
```diff
@@ -46,6 +46,7 @@ def changed():
 
 
 class PeaType(type):
+    """Type of `Pea`, metaclass of `BasePea`."""
```
Comment (nan-wang) [inline jina/peapods/peas/helper.py:49]:
```suggestion
    """Type of :class:`Pea`, metaclass of :class:`BasePea`."""
```

**Review by nan-wang — CHANGES_REQUESTED** [review]

File: jina/peapods/peas/helper.py (hunk 5 above)
Comment (Yongxuanzhang) [reply jina/peapods/peas/helper.py]: Thanks!

**Review by nan-wang — DISMISSED** [review]
LGTM👍

File: jina/peapods/runtimes/jinad/client.py (hunk 3 above)
Comment (Yongxuanzhang) [reply jina/peapods/runtimes/jinad/client.py]: Done

File: jina/peapods/runtimes/zmq/zed.py (hunk 4 above)
Comment (Yongxuanzhang) [reply jina/peapods/runtimes/zmq/zed.py:134]: Done

**Review by theUnkownName — DISMISSED** [review]
Great job!

**Review by theUnkownName — APPROVED** [review]

**Pull request merged**